$OpenBSD: patch-src_gui_main_cpp,v 1.5 2012/08/25 13:30:50 ajacoutot Exp $

gui/main.cpp:37:22: error: revision.h: No such file or directory

--- src/gui/main.cpp.orig	Sat Aug 25 15:18:11 2012
+++ src/gui/main.cpp	Sat Aug 25 15:18:16 2012
@@ -34,7 +34,6 @@
  *   description          : This file contains the main method.
  */
 
-#include "revision.h"
 #include "config.h"
 #include "common.h"
 
